core/loadpoint.go

```diff
@@ -1071,11 +1069,7 @@ func (lp *Loadpoint) pvMaxCurrent(mode api.ChargeMode, sitePower float64, batter
 	// switch phases up/down
 	if _, ok := lp.charger.(api.PhaseSwitcher); ok {
 		availablePower := -sitePower + lp.chargePower
-
-		// in case of scaling, keep charger disabled for this cycle
-		if lp.pvScalePhases(availablePower, minCurrent, maxCurrent) {
-			return 0
-		}
+		_ = lp.pvScalePhases(availablePower, minCurrent, maxCurrent)
```

Comment:
@MarkusGH ich stolpere bei https://github.com/evcc-io/evcc/issues/9458 grad über diese Änderung. In dem Moment wo wir 1p3p machen ist doch weiteres Justieren des Stromes nutzlos- wäre hier nicht trotzdem der early exit notwendig, im Zweifel mit `minCurrent` als Ergebnis?

Reply:
Der alte Zustand war richtig. Der Gedanke, war und ist den Charger die Phasenumschaltung und die Ladestromregelung in evcc den Strom festlegen zu lassen - Regelung nach Messwerten. Der early Exit hier ist da unnötig bis kontraproduktiv, denn je nach Reaktionszeit des Chargers hat man dann für eine gewisse Zeit den "falschen" Strom.